syntax = "proto3";

package stream;

service NiftyService {
    //an example of getting a feature
    rpc NiftyCall (NiftyRequest) returns (stream NiftyResponse) {}
}

// The request containing the start and end dates
message NiftyRequest {
  int64 startDate = 1;
  int64 endDate = 2;
}

// The response containing the message
message NiftyResponse {
  int64 date = 1;
  double value = 2;
}
